### ISSUE TITLE
Lack of DVGO

### ISSUE THREAD

**Issue by Unrealluver** [openxrlab/xrnerf#12]
Greetings,

Thanks for your awesome work! May I know if you would add DVGO to this project soon?

**Comment by daili0015** [comment]
Thanks for your attention to XRNerf ! 
For DVGO, do you mean this method: https://github.com/sunset1995/DirectVoxGO ?

**Comment by Unrealluver** [comment]
> 

Yes, I mean the DirectVoxGO, would you like to add this method further?

**Comment by daili0015** [comment]
We value community requests and suggestions, this has been add to [wishlist](https://github.com/openxrlab/xrnerf/discussions/11#discussioncomment-3569762). We may schedule based on community response.
